content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -424,10 +423,26 @@ sudo systemctl daemon-reload
 sudo systemctl start crio
 ```
 
-Refer to the [CRI-O installation guide](https://github.com/kubernetes-sigs/cri-o#getting-started)
+Refer to the [CRI-O installation guide](https://github.com/cri-o/cri-o/blob/master/install.md)
 for more information.
 
 
+#### cgroup driver
+
+CRI-O uses the systemd cgroup driver per default. To switch to the `cgroupgfs`
```

Comment:
Did you mean:
```suggestion
CRI-O uses the systemd cgroup driver per default. To switch to the `cgroupfs`
```
?

Reply:
Yes this is a typo, fixed as suggested :+1: